webextensions/manifest/action.json

```diff
@@ -3,25 +3,22 @@
     "manifest": {
       "action": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action",
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action",
           "support": {
             "chrome": {
               "version_added": "88",
               "notes": [
                 "Available for use in Manifest V3 or later.",
-                "If an extension defines a browser action, it is not allowed to define a page action as well."
+                "If an extension defines an action, it is not allowed to define a page action as well."
```

Comment:
@Rob--W correct?

Reply:
Chrome does not support `page_action` in MV3 at all. So "not allowed to define a page action" is not conditional on the action key being present. In MV2, Chrome does not allow browser_action and page_action to co-exist (and doesn't support the action key either).

FYI: Safari does not support page_action nor browser_action in MV3 either (only `action`). In MV2 Safari does support page_action and browser_action.

(and for completeness: Firefox supports browser_action only in MV2, action only in MV3, and page_action in MV2+MV3).

Reply:
Thanks @Rob--W I've removed this note.